lib/api/core/entrypoints/manifest.js

```diff
@@ -19,9 +19,7 @@
  * limitations under the License.
```

Comment:
Shouldn't we get ride of this file ? I don't see the point since it's almost empty

Reply:
It inherits AbstractManifest, which is also implemented by https://github.com/kuzzleio/kuzzle/blob/manifest-files-are-required/lib/api/core/plugins/manifest.js

I think it's best to keep this implementation that way: there are always slight differences between protocols and plugins manifest, and something tells me that the differences will only increase in the future.

Reply:
Ok I see! Maybe we can move this file to a more explicit directory ? (Maybe we need a `protocols` directory?)

Reply:
Sounds like a good idea.